### DISCUSSION TITLE
What's the meaning "Bypassing path ranking (e.g. to display a)" in advantages of Named Routes? 命名路由优点中的「绕过路径排序」是什么意思？

### DISCUSSION THREAD

**Discussion by trojanyao** [vuejs/router#1761] [Help and Questions] [answered]
只检索到两个相关的帖子，都没有说明白。我想，要弄明白这个，首先得弄清楚「路径排序」Path Ranking。所以这个是什么意思呢？

[Why using name with router-link is better than just using to with path?](https://stackoverflow.com/questions/74254375/why-using-name-with-router-link-is-better-than-just-using-to-with-path)

[1x3 VueRouter - 基础 - 命名路由 & 命名视图](https://juejin.cn/post/6999254861409681421#heading-1)

**Comment by posva** [comment, marked as answer]
A named route is resolved by name, skipping any comparison with route records while a regular string is resolved by comparison.

Sometimes [routes can match the same paths](https://paths.esm.dev/?p=AAMsIPQgYAEL9lNgQAECUgPgDIFiDtW1F4AQadkJACA.#)

**Reply by trojanyao** [reply]
Thanks!
